examples/pretrained_imagenet_scivision_example.ipynb

```diff
@@ -22,9 +29,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%load_ext autoreload\n",
-    "%autoreload 2\n",
-    "%matplotlib inline"
+    "from scivision.catalog import default_catalog\n",
+    "from scivision.io import load_pretrained_model"
```

Comment:
May be a separate PR, but perhaps we should make this even simpler if possible: `from scivision import default_catalog, load_pretrained_model`

Reply:
Yes deliberately omitted. Maybe there is an argument for a separate `io` and `catalog` modules, though.

Reply:
seems like it's good to keep the modules separate, but there is no reason why `default_catalog` and `load_pretrained_model` couldn't be imported in the scivision `__init__.py`? Then it's even simpler:

```python
from scivision import default_catalog, load_pretrained_model
```

Reply:
Good point!